stages/org.osbuild.rpm

```diff
@@ -47,16 +45,47 @@ SCHEMA = """
     "description": "Array of RPM content hashes",
     "type": "array",
     "items": {
-      "type": "string",
-      "description": ".rpm file checksum, prefixed with 'md5:', 'sha1:', 'sha256:', 'sha384:', or 'sha512:', indicating the algorithm used."
+      "oneOf": [
+        {
+          "type": "string",
+          "description": ".rpm file checksum, prefixed with 'md5:', 'sha1:', 'sha256:', 'sha384:', or 'sha512:', indicating the algorithm used."
+        },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["checksum"],
+          "properties": {
+            "checksum": {
+              "type": "string",
+              "description": ".rpm file checksum, prefixed with 'md5:', 'sha1:', 'sha256:', 'sha384:', or 'sha512:', indicating the algorithm used."
+            },
```

Comment:
Could extract that into a definition to not have to repeat yourself. Can do later though.

Reply:
Ah, yes, of course. Thanks. Should fix this and the other in a follow-up.